README.md

```diff
@@ -1,46 +1,90 @@
+reliable workflows. It is based on [Temporal](https://temporal.io/), hiding
+many of its infrastructure and coding complexities.
+
+AutoKitteh can be self-hosted, and has a cloud offering as well.
+
+Once installed, AutoKitteh is a scalable "serverless" platform (with batteries
+included) for DevOps, FinOps, MLOps, SOAR, productivity tasks, critical
+backend business processes, and more.
+
+<br/>
+
+![GitHub License](https://img.shields.io/github/license/autokitteh/autokitteh)
 [![Go Reference](https://pkg.go.dev/badge/go.autokitteh.dev/autokitteh.svg)](https://pkg.go.dev/go.autokitteh.dev/autokitteh)
 [![Go Report Card](https://goreportcard.com/badge/go.autokitteh.dev/autokitteh)](https://goreportcard.com/report/go.autokitteh.dev/autokitteh)
 
-# autokitteh
+[![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/autokitteh/autokitteh)](https://github.com/autokitteh/autokitteh/commits/main)
+[![CI Status](https://github.com/autokitteh/autokitteh/actions/workflows/ci-go.yml/badge.svg)](https://github.com/autokitteh/autokitteh/actions)
+
+[![YouTube Channel](https://img.shields.io/badge/autokitteh-ff0000?logo=youtube)](https://www.youtube.com/@autokitteh-mo5sb)
+[![LinkedIn](https://img.shields.io/badge/autokitteh-0e76a8?logo=linkedin)](https://www.linkedin.com/company/autokitteh/posts/?feedView=all)
+
+</div>
+
+## High-Level Architecture
 
-autokitteh is an open-source platform for developing and managing automated,
-reliable, durable, long-running workflows with simple tools and familiar
-languages.
+![Architecture diagram](/docs/images/architecture.png)
 
-It is a developer-first alternative to no-code/low-code platforms (such as Zapier, make.com, etc.) and a durable execution platform (a complement to Temporal). It offers tools and a simplified abstraction for crafting reliable, long-running workflows without sacrificing the power and flexibility of direct code manipulation.
+**Platform:** A scalable server that provides interfaces for building projects
+(workflows), deploying them, triggering the code with webhooks or schedulers,
+executing the code as durable workflows, and managing these workflows.
 
-autokitteh promotes a developer-first approach, catering to both inexperienced
-beginners and busy experts, with a wide variety of skill sets and use cases:
+**API:** AutoKitteh is an API-first platform. All services are available via
+gPRC / HTTP.
 
-- CI/CD pipelines and DevOps processes
-- Infrastructure and backend systems orchestration
-- IT, ops, and cybersecurity SOAR runbooks
-- Cross-system syncs and integrations
-- Sales, marketing, and back-office automations
+**Built-in integrations:** Slack, GitHub, Twilio, ChatGPT, Gemini, Gmail,
+Calendar, HTTP, gRPC and many more. It's easy to add new integrations.
 
-autokitteh hides away the toil and provides advanced engineering features
-out-of-the-box:
+**Supported programming languages:** Python, Starlark (a dialect of Python),
+and TypeScript (coming soon).
```

Comment:
IMO we should use `JavaScript` and not `TypeScript` since the term is bigger. 
@RonenMars What do you think?

Reply:
Hey, `TypeScript` is eventually `JavaScript`, meaning we probably can run both.
Regarding the terms I think that TypeScript is bigger, since it contains both JS and TS.

Reply:
I thought TS is a more popular choice than "regular" JS nowadays, but I'm certainly not the expert. Done.